packages/story-editor/src/components/canvas/karma/pageMenuActions.karma.js

```diff
@@ -110,4 +110,43 @@ describe('PageMenu integration', () => {
       });
     });
   });
+
+  it('should render selection frame handles below page menu but allow click-through', async () => {
+    // Add image
+    const mediaItem = fixture.editor.library.media.item(0);
+    await fixture.events.mouse.clickOn(mediaItem, 20, 20);
+
+    // Press right-arrow 10 times to move it into correct location
+    await Array.from(Array(10)).reduce(
+      (p) => p.then(() => fixture.events.keyboard.press('right')),
+      Promise.resolve()
+    );
+
+    // Snapshot it
+    await fixture.snapshot();
```

Comment:
Is there any rule of thumb for taking snapshots? Here the broken and fixed state looks exactly the same (until you hover it).

Reply:
I just wanted a snapshot of the frame being behind the page menu buttons. Wasn't sure if we had that. This test would fail too if that was changed at some point in the future, but the adds a visual "confirmation" if that bug occurred.